### PR TITLE
Set path for custom application on Windows

### LINKED ISSUE #1614: Custom Python Applications in Windows require multiple additions to PATH
On Windows 7x64 SP1, I got the following error when following the instructions [here](http://docs.qgis.org/2.14/en/docs/pyqgis_developer_cookbook/intro.html#running-custom-applications) for developing a custom Python app, where `from qgis._core import *` produces:
```shell
ImportError: DLL load failed: The specified module could not be found
```
I accidentally raised this as an [issue on the software by accident](http://hub.qgis.org/issues/16042) before understanding the problem with the error. I needed to add two different `bin` folders to the `PATH` variable:
```
C:\Program Files\QGIS Essen\apps\qgis-ltr\bin
C:\Program Files\QGIS Essen\bin
```
I wanted to confirm whether this affects other QGIS/Windows versions before editing the documentation.

### PR DESCRIPTION
**Pull request by DelazJ**
Fixes #1614

### CODE REVIEW

File: source/docs/pyqgis_developer_cookbook/intro.rst [hunk 1]
```diff
@@ -295,7 +295,9 @@ Fix this by adding the directories where the QGIS libraries reside to search
 path of the dynamic linker:
 
 * on Linux: :command:`export LD_LIBRARY_PATH=/qgispath/lib`
-* on Windows: :command:`set PATH=C:\\qgispath;%PATH%`
+* on Windows: :command:`set PATH=C:\\qgispath\\bin;C:\\qgispath\\apps\\qgisrelease\\bin;%PATH%`
+  where ``qgisrelease`` should be replaced with the type of release you are
```
[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/pyqgis_developer_cookbook/intro.rst]: @yjacolin looks like there's a missing word in your comment. Not sure to understand your style yet.

[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/pyqgis_developer_cookbook/intro.rst]: Ok I see. Do you think that it should also be applied to the `qgispath` "variable"?

[the remark this answers is not in the document]
Comment (yjacolin) [reply source/docs/pyqgis_developer_cookbook/intro.rst]: oh yes! I think so.

[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/pyqgis_developer_cookbook/intro.rst]: Done but looks a bit ugly :)

[the remark this answers is not in the document]
Comment (yjacolin) [reply source/docs/pyqgis_developer_cookbook/intro.rst]: probably but reader can find easily parameter to be changed. ``\\`` are not better :)

**Pull request merged**